CodeStarterCamp_Week4/body.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  body.swift
+//  CodeStarterCamp_Week4
+//
+//  Created by minsong kim on 2023/03/13.
+//
+
+import Foundation
+
+class BodyCondition {
+    var upperBodyStrength: Int
+    var lowerBodyStrength: Int
+    var muscularEndurance: Int
+    var fatigability: Int
+    
+    init(upperBodyStrength: Int, lowerBodyStrength: Int, muscularEndurance: Int, fatigability: Int) {
+        self.upperBodyStrength = upperBodyStrength
+        self.lowerBodyStrength = lowerBodyStrength
+        self.muscularEndurance = muscularEndurance
+        self.fatigability = fatigability
+    }
+    
+    func conditionCheck() {
+        print(
+        """
+        --------------
+        현재의 컨디션은 다음과 같습니다.
+        상체근력: \(upperBodyStrength)
+        하체근력: \(lowerBodyStrength)
+        근지구력: \(muscularEndurance)
+        피로도: \(fatigability)
+        --------------
+        """
+        )
+    }
+}
+
+struct Activity {
+    let name: String
+    let action: (BodyCondition) -> Void
+}
+
+
+let 윗몸일으키기: Activity = Activity(name: "윗몸일으키기", action: { BodyCondition -> Void in
```

Comment:
클로저는 리턴타입을 생략할 수 있습니다! ㅎㅎ
클로저의 경량 문법, 트레일링 클로저에 대해 한번 자세히 공부해보세요! 처음에는 조금 이해하기 어려울 수 있지만 잘 활용하면 가독성이 훨씬 좋아집니다!
그리고 클로저의 내부 파라미터도 로우카멜케이스를 사용해야 합니다! 😁

```suggestion
let 윗몸일으키기: Activity = Activity(name: "윗몸일으키기", action: { bodyCondition in
```